pkg/frontend/frontend.go

```diff
@@ -289,27 +291,73 @@ func (f *frontend) Run(ctx context.Context, stop <-chan struct{}, done chan<- st
 	}
 }
 
+func adminJmespathFilterValidate(filter string) (*jmespath.JMESPath, error) {
+	if filter == "" {
+		return nil, nil
+	}
+	return jmespath.Compile(filter)
```

Comment:
suggest returning an *api.CloudError here then no changes needed to adminReply?

Reply:
done